custom_components/oncharger/translations/en.json

```diff
@@ -68,6 +72,15 @@
       "current": {
         "name": "Current"
       },
+      "current1": {
```

Comment:
I'd prefer this approach rather:

https://github.com/krasnoukhov/homeassistant-smart-maic/blob/b965e7934f88665fc5b0a9b5f587a126c3cc6038/custom_components/smart_maic/entity.py#L67